code/src/lib/Containers.js

```diff
@@ -0,0 +1,116 @@
+import styled from 'styled-components/macro'
+
+export const Main = styled.main`
+  display: grid;
+  min-height: 100vh;
+  grid-template-rows: min-content;
+  justify-content: center;
```

Comment:
I like how you organize the styled-components inside the lib folder and export them when needed.